### PR TITLE
User account creation something else

### PR DESCRIPTION
**Pull request by creativeappdevamit**
# What and why?

# How to test?

# Trello
https://trello.com/c/XQ2gnUbP/369-s37-user-account-creation-something-else

### CODE REVIEW

File: secure_message/validation/domain.py [hunk 1]
```diff
@@ -141,7 +149,15 @@ def validate_non_zero_field_length(self, field_name, length, max_field_len):
     def validate_field_length(field_name, length, max_field_len, data=None):
         if length > max_field_len:
             logger.info('Field is too large', field_name=field_name, length=length, max_field_len=max_field_len)
-            raise ValidationError(f'{field_name} field length must not be greater than {max_field_len}', field_name, [], data)
+            raise ValidationError(f'{field_name} field length must not be greater than {max_field_len}', field_name, [],
+                                  data)
+
+    @staticmethod
+    def validate_category_type(field_name, data=None):
+        if data in ['SURVEY', 'TECHNICAL', 'MISC']:
```
Comment (mfcrocker) [inline secure_message/validation/domain.py:157]: Not sure if this is what you meant?

```suggestion
        if data not in ['SURVEY', 'TECHNICAL', 'MISC']:
```

**Review by mfcrocker — APPROVED** [review]
LGTM but one thing looks weird to me

File: secure_message/validation/domain.py (hunk 1 above)
Comment (creativeappdevamit) [reply secure_message/validation/domain.py:157]: no actually I wanted to check if the provided data against category matches these three attributes.

**Pull request merged**